Feature/no01 css

beforeisland opened the pull request:
NRI模写、CSSでの装飾が完了しました。

beforeisland committed:
fix: ヘッダー部分の装飾実装

ヘッダー部分のサイトロゴ、グローバルメニューのfloat・hover時の挙動、サブメニューのfloat、パンくずリストを実装しました。

beforeisland committed:
fix: ヘッダーメニューのcss実装

サイトロゴ、グローバルメニューの装飾・float・hover時の挙動、サイドナビの装飾・float・hover時の挙動、メニューの固定、パンくずリストの実装を行いました。

beforeisland committed:
Merge branch 'feature/no01-css' of https://github.com/beforeisland/ryosukemaejima into feature/no01-css

# Conflicts:
#	html-nri/common/css/style.css

beforeisland committed:
fix: ヘッダーメニューのコンフリクトの回避

リモートからプルしたところコンフリクトが起こり上書きされてしまったため、ローカルの変更を正として回避しました。

beforeisland committed:
add: メインビジュアルの装飾実装

テキストの装飾、画像とテキストの重なり実装、ヘッダーメニューの重なり順を上げるz-index実装。

beforeisland committed:
add: snsリンクAタイプ、メイン見出し装飾実装

snsリンク部分の装飾・float調整、メイン記事部分のh2装飾・背景画像アイコン設定

beforeisland committed:
add: メイン記事装飾、ボタンの実装

メイン記事の装飾及び、pdf用のボタンの装飾・アイコン埋め込み・ヘッドアイコン「>」埋め込み・opacity実装。

beforeisland committed:
add: メイン記事下の左右リンク装飾

メイン記事下の左右リンクのfloat・hover時の挙動・装飾実装

beforeisland committed:
add: メイン記事下6記事の装飾実装

メイン記事下6記事の装飾・float・hover時の挙動実装

beforeisland committed:
add: メイン記事・企業情報の見出し装飾

メイン記事・企業情報の見出し装飾の実装。

beforeisland committed:
add: メイン記事・企業情報のリンク装飾実装

メイン記事・企業情報のリンクの装飾・border・float・頭アイコン・調整実装

beforeisland committed:
add: メイン記事、前のページへ戻る装飾

メイン記事の前のページへ戻る部分の装飾を実装しました。

beforeisland committed:
add: メイン記事下のSNSリンク装飾実装

メイン記事下のSNSリンクの装飾・float・調整の実装

beforeisland committed:
add: サブ記事リンク部分の装飾

サブ記事のリンク部分の装飾・float・hover時の挙動・頭のアイコン・各種調整を実施

beforeisland committed:
add: フッター装飾実装

フッターの装飾・リンクfloat・装飾・コピーライト実装

beforeisland committed:
add: 右下トップへ戻るボタン実装

右下にあるトップへ戻るボタンを実装

beforeisland committed:
fix: 全体見直し

googleフォントの導入、pdfボタン部分のline-height調整、ヘッダーメニューpdding調整

piasoratech reviewed (changes requested):
最大画面で表示した際にヘッダーの幅が短くなってしまっているので、幅が変動してサイズが合うようにしてください。
![image](https://user-images.githubusercontent.com/57445698/87874113-3a97e580-ca02-11ea-9c75-b74ff99b1420.png)

beforeisland committed:
fix: ヘッダー固定メニュー修正

ブラウザ画面幅を広げたときに、ヘッダー固定メニューの横幅が切れてしまう問題を修正しました。htmlにてヘッダーメニューを囲うようにdivを準備し、外側のdivの横幅を無制限とし、内側に格納したdivの横幅を指定することで実装。

beforeisland commented:
> 最大画面で表示した際にヘッダーの幅が短くなってしまっているので、幅が変動してサイズが合うようにしてください。
> ![image](https://user-images.githubusercontent.com/57445698/87874113-3a97e580-ca02-11ea-9c75-b74ff99b1420.png)

修正完了

fix: ヘッダー固定メニュー修正	a455ad6	beforeisland <beforeisland@gmail.com>	2020/07/19 22:27

piasoratech reviewed (changes requested):
全体的な指摘になります。
- 画像ファイルのサイズ指定はhtml側で行わずcss側で行うようにしてください。見た目に関することはcssにまとめるのが原則です。
- id名やclass名の命名規則がケバブケースとキャメルケースが混在しているので統一をしましょう。
- マジックナンバー(xxxx1,xxxxx2)での命名がされている部分があるので、修正してください。名前は見ただけでどんな全体的な指摘になります。
- 画像ファイルのサイズ指定はhtml側で行わずcss側で行うようにしてください。見た目に関することはcssにまとめるのが原則です。
- id名やclass名の命名規則がケバブケースとキャメルケースが混在しているので統一をしましょう。
- マジックナンバー(xxxx1,xxxxx2)での命名がされている部分があるので、修正してください。名前は見ただけでどんな内容なのかわかるようにしておく必要があります。

beforeisland committed:
update: id,class命名規則の変更

id,classの命名規則をケバブケースに統一しました。

beforeisland committed:
update: マジックナンバー付きclass名の修正及び装飾方法の変更

マジックナンバーで指定していたclass名(articleDetail-Text1及び2)を修正、統一(article-deatil-text)。この2要素をdiv要素に格納し、nth-childで装飾先を指定する方法に変更。

beforeisland committed:
update: 画像のサイズ指定をhtml -> CSSへ変更

アイコンなどの画像のサイズ指定を、htmlのimgタグwidth要素、height要素で指定していたものを、CSSで指定する方法に変更。

beforeisland commented:
> 全体的な指摘になります。
> 
> * 画像ファイルのサイズ指定はhtml側で行わずcss側で行うようにしてください。見た目に関することはcssにまとめるのが原則です。
> * id名やclass名の命名規則がケバブケースとキャメルケースが混在しているので統一をしましょう。
> * マジックナンバー(xxxx1,xxxxx2)での命名がされている部分があるので、修正してください。名前は見ただけでどんな全体的な指摘になります。
> * 画像ファイルのサイズ指定はhtml側で行わずcss側で行うようにしてください。見た目に関することはcssにまとめるのが原則です。
> * id名やclass名の命名規則がケバブケースとキャメルケースが混在しているので統一をしましょう。
> * マジックナンバー(xxxx1,xxxxx2)での命名がされている部分があるので、修正してください。名前は見ただけでどんな内容なのかわかるようにしておく必要があります。

下記コミットにて修正完了しました。

update: id,class命名規則の変更	66ee518	beforeisland <beforeisland@gmail.com>	2020/07/20 21:08

update: マジックナンバー付きclass名の修正及び装飾方法の変更	3a1a889	beforeisland <beforeisland@gmail.com>	2020/07/20 21:13

update: 画像のサイズ指定をhtml -> CSSへ変更	f2b8dce	beforeisland <beforeisland@gmail.com>	2020/07/20 21:30

choimake commented on html-nri/common/css/style.css:
```diff
@@ -1,47 +1,718 @@
+    margin-right: 0;
+}
+
+#page-body-main .page-body-main-container .article-navi ul li:nth-child(1), {
+    height: 70px;
+}
```
ここにシンタックスエラーがあるので、チェッカーで確認して修正してください。

beforeisland replied:
修正しました

fix: CSSシンタックスエラー修正	9cf1d5b	beforeisland <beforeisland@gmail.com>	2020/07/22 22:14

choimake reviewed (commented):
確認しました。最初に比べて格段によくなっているし、レイアウトもかなりできています。

いくつか修正して欲しい点がありますので、そちら対応お願いします。

- ヘッダーのUIの位置がオリジナルとくらべて中央に寄り過ぎているように見えます。（全画面）
提出物
<img width="801" alt="スクリーンショット 2020-07-21 9 15 20" src="https://user-images.githubusercontent.com/52233450/87998892-3370f480-cb34-11ea-817f-46ee54364456.png">
オリジナル
<img width="1426" alt="スクリーンショット 2020-07-21 9 15 41" src="https://user-images.githubusercontent.com/52233450/87998901-38ce3f00-cb34-11ea-8e5e-122e3392edc7.png">

- 全画面でみたときにやや余白が小さくみえます。
- 太字のフォントがオリジナルとくらべてやや薄いように見えます。

提出物
<img width="1418" alt="スクリーンショット 2020-07-21 9 16 49" src="https://user-images.githubusercontent.com/52233450/87999225-256fa380-cb35-11ea-8e05-2e438402e1ae.png">
オリジナル
<img width="1407" alt="スクリーンショット 2020-07-21 9 17 02" src="https://user-images.githubusercontent.com/52233450/87999220-1f79c280-cb35-11ea-867a-a580f81feab8.png">

- copyrightはpタグで囲む必要はないと思います(構造的に段落ではないのでは？）。スタイル当てるなら、smallタグにstyleあてれば良いかなと

beforeisland committed:
fix: 固定ヘッダーの装飾、中央寄り修正

ヘッダーメニューを囲うdivのwidth:1250pxをmin-widthに変更し、ブラウザの幅に沿ってアイコンが広がるように変更。グローバルナビを右寄せにするため、グローバルナビ・サイドナビを囲う親要素となるdivを用意しfloat:rightを指定して実装。

beforeisland committed:
fix: サブ記事リンク部分の装飾修正

太字部分のフォントの薄さはgoogleフォントのNote sans JP "700"が入っていなかったことによるため、インストールして対応。オリジナルと比べてフルサイズ時の余白が足りないため、4連floatの幅を25%とし、ブラウザサイズによって幅を可変させるように変更。また、親要素にmax-widthとmin-widthを指定、子要素にmin-widthを指定することでブラウザを縮小した際の表示崩れを防止した。

beforeisland committed:
fix: コピーライト表記のタグ修正

ページの文書構造的にpタグ不要と判断し削除。合わせてスタイル修正。

beforeisland committed:
fix: CSSシンタックスエラー修正

要素の最後に「,」が入っていたため削除。

beforeisland commented:
> 確認しました。最初に比べて格段によくなっているし、レイアウトもかなりできています。
> 
> いくつか修正して欲しい点がありますので、そちら対応お願いします。
> 
> * ヘッダーのUIの位置がオリジナルとくらべて中央に寄り過ぎているように見えます。（全画面）
>   提出物
> 
> <img alt="スクリーンショット 2020-07-21 9 15 20" width="801" src="https://user-images.githubusercontent.com/52233450/87998892-3370f480-cb34-11ea-817f-46ee54364456.png">
> 
> オリジナル
> <img alt="スクリーンショット 2020-07-21 9 15 41" width="1426" src="https://user-images.githubusercontent.com/52233450/87998901-38ce3f00-cb34-11ea-8e5e-122e3392edc7.png">
> 
> * 全画面でみたときにやや余白が小さくみえます。
> * 太字のフォントがオリジナルとくらべてやや薄いように見えます。
> 
> 提出物
> <img alt="スクリーンショット 2020-07-21 9 16 49" width="1418" src="https://user-images.githubusercontent.com/52233450/87999225-256fa380-cb35-11ea-8e05-2e438402e1ae.png">
> オリジナル
> <img alt="スクリーンショット 2020-07-21 9 17 02" width="1407" src="https://user-images.githubusercontent.com/52233450/87999220-1f79c280-cb35-11ea-867a-a580f81feab8.png">
> 
> * copyrightはpタグで囲む必要はないと思います(構造的に段落ではないのでは？）。スタイル当てるなら、smallタグにstyleあてれば良いかなと

下記にて修正しました。
fix: 固定ヘッダーの装飾、中央寄り修正	a8d9c4f	beforeisland <beforeisland@gmail.com>	2020/07/22 20:59
fix: サブ記事リンク部分の装飾修正	785de8d	beforeisland <beforeisland@gmail.com>	2020/07/22 21:59
fix: コピーライト表記のタグ修正	4ca4cac	beforeisland <beforeisland@gmail.com>	2020/07/22 22:12

choimake commented on html-nri/index.html:
```diff
@@ -316,10 +332,10 @@ <h3>事業案内</h3>
     </footer>
-    <div class="pagetop">
-        <p><a href="#pageHead">&and;</a></p>
+    <div class="page-top-icon">
+        <p><a href="#page-head">&and;</a></p>
     </div>
```
すいません。これ見落としてました。(本来はhtmlの段階で修正しておくべき)
page-top-icon関連のタグ、これはfooterタグ内に配置してください。

beforeisland replied:
修正完了

fix: 右下アイコン周りのhtml構造修正

choimake reviewed (commented):
コメントしましたので確認お願いします。

beforeisland committed:
fix: 右下アイコン周りのhtml構造修正

右下のトップへ戻るアイコンをfooter内に格納しました。

choimake commented:
LGTMです。次に進んでください。
